app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,26 @@
  *= require_tree .
  *= require_self
  */
+
+ .topnav {
+   background-color: #333;
+   overflow: hidden;
+ }
+
+ .topnav a {
+   float: left;
+   color: #f2f2f2;
+   text-align: center;
+   padding: 14px 16px;
+   font-size: 17px;
+ }
+
+ .topnav a:hover {
+   background-color: #ddd;
+   color: black;
+ }
+ /* This isn't working */
+ .topnav a.active {
```

Comment:
So there's a hover color-change here that's not working? we can fix that I'm sure